share/change.py

```diff
@@ -292,18 +292,30 @@ def change(self):
         # Hacky fix for SHARE-604
         # If the given date_updated is older than the current one, don't accept any changes that would overwrite newer changes
         ignore_attrs = set()
-        if issubclass(self.model, apps.get_model('share', 'creativework')) and 'date_updated' in self.attrs and self.instance.date_updated:
-            date_updated = pendulum.parse(self.attrs['date_updated'])
```

Comment:
Not that I know of? The extra check for `self.instance` can be removed. I forgot about the early return on line 274.